### DISCUSSION TITLE
Display which node is effective in sending data

### DISCUSSION THREAD

**Discussion by Kurt-AZ** [nextgenhealthcare/connect#5735] [Feature/Enhancement Ideas]
We are using the advanced clustering plugin (version 4.0.1) and have noticed that it is not always the Mirth node which has the polling role, that is effectively **sending** out the data.
In certain circumstances, it can be that another node is sending out the data to its destination.
For the moment this can nowhere be seen in the interface (GUI) which node is effectively sending the data.
Only the node that has _received_ the data can be checked in the GUI (column _Original Server ID_).

If a certain node has to be rebooted or is failing, it would be helpful to check in advance if the node is sending out some data.

**Comment by pacmano1** [comment]
@Kurt-AZ - almost sounds like a bug - maybe throw it in the issues/features section?

**Reply by Kurt-AZ** [reply]
Hi @pacmano1 

When using (and paying) for the advanced clustering plugin, I find it indeed a pitty that this can not be seen in the Mirth interface.
We just noticed it by coincidence (through a Wireshark session) that the node that did not have the polling role was sending out data.
For the same channel, the polling node was receiving the data. So in our case, both nodes were handling the data.
So this situation (2 nodes handling data), without knowing/seeing this in the interface (GUI) can be very confusing.

We already created a case @Nextgen were they admitted that this was not foreseen in the interface

> After some further review, it appears that there is no setting that can be used to keep the polling node to start sending the data. Correct, we would need to stop the Mirth Connect service on the sending node to have the other node take over.
Additionally, there is unfortunately no areas that display which node is effective in sending the data. I would recommend putting this idea towards the Community Idea board.

Meanwhile, as suggested by you, I created a new idea in the issues section: [#5741](https://github.com/nextgenhealthcare/connect/issues/5741)
